specs/Energy/Battery/example.jsonld

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "urn:ngsi-ld:Battery:santander:energy:bat-ac-d95372df39",
+  "type": "Battery",
+  "source": {
```

Comment:
check the NGSI-LD representation of other entities, this is totally incorrect

Reply:
ok